javascript/ql/src/semmle/javascript/security/dataflow/ClientSideUrlRedirectCustomizations.qll

```diff
@@ -132,6 +132,15 @@ module ClientSideUrlRedirect {
     }
   }
 
+  /**
+   * An argument to `importScripts(..)` - which is used inside `WebWorker`s to import new scripts - viewed as a `ScriptUrlSink`.
+   */
+  class ImportScriptsSink extends ScriptUrlSink {
+    ImportScriptsSink() {
+      this = DataFlow::globalVarRef("importScripts").getACall().getAnArgument()
```

Comment:
> importScripts() and self.importScripts() are effectively equivalent — both represent importScripts() being called from inside the worker's inner scope.
> (https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts)

Do we support the mentioned `self.importScripts` as well?

Reply:
We should - I'll add a test for it.